boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -153,11 +153,30 @@ go-generate:
 	${GOENV} go generate $(TESTTARGETS)
 	# Don't forget to commit generated files
 
+# go-get-tool will 'go install' any package $2 and install it to $1.
+define go-get-tool
+@[ -f $(1) ] || { \
+set -e ;\
+TMP_DIR=$$(mktemp -d) ;\
+cd $$TMP_DIR ;\
+go mod init tmp ;\
+echo "Downloading $(2)" ;\
+GOBIN=${PWD}/bin go install $(2) ;\
```

Comment:
Looks like this will ignore $1 and install into $TMP_DIR/bin. Are we missing a step to copy it out to $1?

Reply:
updated